lte/gateway/c/core/oai/tasks/grpc_service/AmfServiceImpl.cpp

```diff
@@ -67,36 +67,58 @@ Status AmfServiceImpl::SetAmfNotification(ServerContext* context,
   // ToDo processing ITTI,ZMQ
 
   itti_n11_received_notification_t itti_msg;
```

Comment:
initialize itti_msg and pass as a reference to SetAmfNotification_itti

Reply:
Comment addressed.